### PR TITLE
Add Quality Declaration [9390]

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Add Fast CDR Quality Declaration
Update README to link to the Quality Declaration

For more info about Fast CDR Platform support, please refer to #84

For more info about the code style, please refer to eProsima/cpp-style#3

For more info about CONTRIBUTING, please refer to eProsima/policies#2

For more info about the Fast CDR documentation (API reference), please refer to eProsima/all-docs#17

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/45/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/46/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/48/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/49/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/48/)
* Linux aarch64[Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux%20aarch64/3/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/3/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/49/)

**Review by brawner — APPROVED** [review]
Looks good!

File: QUALITY.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+* Linux-aarch64 [![Linux arm64 ci](http://jenkins.eprosima.com:8080/job/nightly_fastcdr_master_linux_aarch64/badge/icon?subject=%20%20%20Linux-aarch64%20CI%20)](http://jenkins.eprosima.com:8080/view/Nightly/job/nightly_fastcdr_master_linux_aarch64/)
+* Windows [![Windows ci](http://jenkins.eprosima.com:8080/job/nightly_fastcdr_master_windows/label=windows-secure,platform=x64,toolset=v141/badge/icon?subject=%20%20%20%20Windows%20CI%20)](http://jenkins.eprosima.com:8080/job/nightly_fastcdr_master_windows/label=windows-secure,platform=x64,toolset=v141)
+* Mac [![Mac ci](http://jenkins.eprosima.com:8080/job/nightly_fastcdr_master_mac/badge/icon?subject=%20%20%20%20%20%20%20Mac%20CI%20)](http://jenkins.eprosima.com:8080/job/nightly_fastcdr_master_mac)
+
+More information about the supported platforms can be found in [PLATFORM_SUPPORT](PLATFORM_SUPPORT.md)
+
+## Vulnerability Disclosure Policy [7.i]
+
+*eprosima Fast DDS* vulnerability Disclosure Policy can be found [here](https://github.com/eProsima/policies/blob/main/VULNERABILITY.md)
```
Comment (MiguelCompany) [inline QUALITY.md:160]:
```suggestion
**eprosima Fast CDR** vulnerability Disclosure Policy can be found [here](https://github.com/eProsima/policies/blob/main/VULNERABILITY.md)
```

File: QUALITY.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+# Current Status Summary
+
+The chart below compares the requirements in the [REP-2004](https://www.ros.org/reps/rep-2004.html#quality-level-comparison-chart) with the current state of *eprosima Fast DDS*
```
Comment (MiguelCompany) [inline QUALITY.md:164]:
```suggestion
The chart below compares the requirements in the [REP-2004](https://www.ros.org/reps/rep-2004.html#quality-level-comparison-chart) with the current state of **eprosima Fast CDR**
```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]

**Review by MiguelCompany — APPROVED** [review]

**Pull request merged**